### ISSUE TITLE
Update Launch Screen

### ISSUE THREAD

**Issue by workerbee22** [Western-Health-Covid19-Collaboration/wh_covid19_app#279]
When the app starts (known as a cold start), or when bringing the app back to the foreground (this is a warm re-start where the os has re-allocated memory while the app is in the background, requiring it to effectively restart) the app shows a white screen for anywhere from 1 to 4 seconds depending on the speed of the phone before the Home screen or Disclaimer is shown. 

The default launch screen provided by Flutter for both iOS and Android is a plain white screen. Suggest we update the default launch screen (called launch screen in both Android and iOS) with the centered product icon or other design, rather than a white screen. This is not essential but is a key UI polish issue, probably only required for a production release of the app.

**Comment by cjmlgrto** [comment]
Here's the design for that— [Zeplin](https://app.zeplin.io/project/5e7597a68651056664fb6e53/dashboard?seid=5ea4d4ac42591a23572ef3de)

![Launch Screen Image](https://user-images.githubusercontent.com/1712450/80293994-4760c080-87a8-11ea-8f9d-69d9f6c6a63d.png)

Ideally, this screen is built out — not a static image. That way, the logo would resize/position accordingly and align to the logo when the Main screen gets loaded in. In order words, if this screen were actually built out, it should be exactly like the Main screen, but without the rest of the content.

**Comment by lukesleeman** [comment]
Nice work ✨   I've moved this over to ready for dev 🙂

**Comment by workerbee22** [comment]
So just to be clear, this Launch screen is shown by the platform and is before Flutter framework has loaded. So this means we build it twice; once for iOS and once for Android. To 'build it out' we've have to do thi sin native ie. Swift/C# and Java/Kotlin.

Suggest we move out of Version 1 milestone or go for something simpler for Version 1 where we just use an image (vector or PNG). Often the launcher icon is used and centered. Or we can leave it until after Version 1.

**Comment by workerbee22** [comment]
@lukesleeman So I removed 'Ready for Dev' tag for now.

**Comment by workerbee22** [comment]
Like the new government Covid Safe app we should consider the Launch screen along with the Onboarding screens, Disclaimer etc. Because it is all part of app startup and first run ... so we need to make sure this is a completely smooth experience.

**Comment by lukesleeman** [comment]
Yep, this is fine.  This issue isn't part of V1 and can just sit in the backlog until after V1 ships.

With building out the native android version - I used to be pretty good at this, and I've got some ideas on how we can do it and support a bunch of difference screen sizes 😁

Hit me up when we implement!

**Comment by workerbee22** [comment]
Cool ... we can do images easy, but if we want something flashy then its
vector drawables or Java/Kotlin code.

But be aware there are some platform specific issues here too and if we do
something flashy then someone will have to to it in Swift too. 🤣
